web/src/lib/static/admin/findDeadLinks.ts

```diff
@@ -238,29 +265,42 @@ export const findDeadLinks = async (): Promise<Record<string, string[]>> => {
     );
   };
 
-  for (const page of pages) {
-    const promise = new Promise((resolve) => {
-      const htmlUrlChecker = new HtmlUrlChecker(
-        {},
-        {
-          link: (result: any): void => {
-            if (result.broken && !isTemplateLink(result.url.original)) {
-              deadLinks[page].push(result.url.original);
-            }
-          },
-          end: (): void => {
-            resolve({});
-          },
-        }
-      );
-      const url = new URL(process.env.REDIRECT_URL || "");
-      htmlUrlChecker.enqueue(`${url.origin}${page}`, {});
-    });
+  const numberOfBatches = Math.ceil(pages.length / 5);
+
+  for (let j = 0; j < numberOfBatches; j++) {
+    const pagePromises = [];
+    const startIndex = j * 5;
+    const batch = pages.slice(startIndex, startIndex + 5);
+
+    for (const page of batch) {
+      const promise = new Promise((resolve) => {
+        const htmlUrlChecker = new HtmlUrlChecker(
+          {},
+          {
+            link: (result: any): void => {
+              if (
+                result.broken &&
+                !isTemplateLink(result.url.original) &&
+                result.url.original !== "https://twitter.com/i/lists/1148625562895826945"
+              ) {
+                deadLinks[page].push(result.url.original);
+              }
+            },
+            end: (): void => {
+              resolve({});
+            },
+          }
+        );
+        const url = new URL(process.env.REDIRECT_URL || "");
+        htmlUrlChecker.enqueue(`${url.origin}${page}`, {});
+      });
+
+      pagePromises.push(promise);
+    }
 
-    pagePromises.push(promise);
+    await Promise.all(pagePromises);
```

Comment:
Is there a reason we moved this inside the loop?

Reply:
Yeah, so the call here attempts to resolve all the promises simultaneously so previously, this meant every page all at once. 

Now that we're batching it means resolve each batch of 5